Agents/BuildingMatchingAgent/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# First stage: build war file
+#==================================================================================================
+FROM maven:3.6-openjdk-8-slim as builder
+
+# Copy all files into root's home, including the source, pom file, ./m2 directory, credentials and config files
+ADD . /root
+
+# Populate settings templates with credentials, repo name
+WORKDIR /root/.m2
+# (Note that | rather than / is used as the sed delimiter, since encrypted passwords can contain the former, but not the latter
```

Comment:
"latter, but not the former)"

Reply:
Addressed by commit [ee3cc25](https://github.com/cambridge-cares/TheWorldAvatar/pull/616/commits/ee3cc255c196a2ed5e1ca514823e3f49596895d9)